examples/.vale.ini

```diff
@@ -0,0 +1,37 @@
+# Core settings
+# =============
+
+# Location of our `styles`
+StylesPath = "../doc/styles"
+
+# The options are `suggestion`, `warning`, or `error` (defaults to “warning”).
+MinAlertLevel = warning
+
+# By default, `code` and `tt` are ignored.
+IgnoredScopes = code, tt
+
+# By default, `script`, `style`, `pre`, and `figure` are ignored.
+SkippedScopes = script, style, pre, figure
+
+# WordTemplate specifies what Vale will consider to be an individual word.
+WordTemplate = \b(?:%s)\b
+
+# List of Packages to be used for our guidelines
+Packages = Google
+
+# Define the Ansys vocabulary
+Vocab = ANSYS
+
+# Section styles
+# ============== 
+[*.{md,rst}]
+
+# Apply the following styles
+BasedOnStyles = Vale, Google
+
+TokenIgnores = (:(func|class|meth|attr|py):`(?:.|\n)*?`)|(<.*>)
+
+# Removing Google-specific rule - Not applicable under some circumstances
+Google.Colons = NO
+Google.Headings = NO

```

Comment:
This needs to be removed as we need to enforce Google headings. Pinging @PipKat for visibility.

```suggestion
```

Reply:
Roger that!